src/configuration/GPG/GnuPG/params.txt

```diff
@@ -0,0 +1,9 @@
+Key-Type: RSA
+Key-Length: 8192  
+Subkey-Type: RSA
+Subkey-Length: 8192
```

Comment:
Probably 4k is fine now. My concern here is the lack of forward secrecy in OpenPGP. Anyone who captures traffic or has access to mail postboxes may sooner or later be able to factorise a private keys primes. If one is able to do that all communication encrypted with this public key is decipherable. My point here is, that it is probably more difficult to do so for a 8k key than a 4k key ans since 8k are technically possible why should one not use it? :-)